x-pack/plugins/observability/public/pages/overview/overview.stories.tsx

```diff
@@ -41,14 +41,20 @@ const sampleAPMIndices = { transaction: 'apm-*' } as ApmIndicesConfig;
 const withCore = makeDecorator({
   name: 'withCore',
   parameterName: 'core',
-  wrapper: (storyFn, context, { options }) => {
+  wrapper: (storyFn, context, { options: { theme, ...options } }) => {
```

Comment:
@elastic/observability-ui Had to spread to remove `theme` because of the way `options` is force-casted into `CoreStart`, as the new `theme` service is in conflict with the `theme` property of `options`

```
export interface OptionsParameter extends Object {
    storySort?: StorySortParameter;
    theme?: {
        base: string;
        brandTitle?: string;
    };
    [key: string]: any;
}
```